openedx/core/djangoapps/user_authn/config/waffle.py

```diff
@@ -0,0 +1,19 @@
+"""
+Waffle flags and switches for user authn.
+"""
+from __future__ import absolute_import
+
+from openedx.core.djangoapps.waffle_utils import WaffleSwitchNamespace
+
+WAFFLE_NAMESPACE = u'user_authn'
+
+# Switches
+# If this switch is enabled then `edx.org` users must be sign in using their `edx.org` google account
+ENABLE_LOGIN_USING_THIRDPARTY_AUTH_ONLY = 'enable_login_using_thirdparty_auth_only'
+
+
+def waffle():
+    """
+    Returns the namespaced, cached, audited Waffle class for user_authn.
+    """
+    return WaffleSwitchNamespace(name=WAFFLE_NAMESPACE, log_prefix=u'UserAuthN: ')
```

Comment:
This could just be `waffle = WaffleSwitchNamespace(name=WAFFLE_NAMESPACE, log_prefix=u'UserAuthN: ')`. There's no reason to make it a function.